vloopback: compiler warning (variable set but not used)

One more warning:
```
gcc -g -O2 -D_REENTRANT -DMOTION_V4L2 -DMOTION_V4L2_OLD -DTYPE_32BIT="int" -DHAVE_BSWAP   -march=native -mtune=native -Wall -DVERSION=\"3.4.1+gite44a5a9\" -Dsysconfdir=\"/usr/local/etc\" -I/home/ubuntu/ffmpeg/out-3.0/include     -c -o video.o video.c
vloopback_motion.c: In function 'v4l_open_vidpipe':
vloopback_motion.c:48:15: warning: variable 'ostatus' set but not used [-Wunused-but-set-variable]
         char *ostatus;
               ^
vloopback_motion.c:44:15: warning: variable 'loop' set but not used [-Wunused-but-set-variable]
         char *loop;
```

--- a/vloopback_motion.c
+++ b/vloopback_motion.c
@@ -41,11 +41,9 @@ static int v4l_open_vidpipe(void)
 
     if (strcmp(minor, "5") < 0) {
         FILE *vloopbacks;
-        char *loop;
         char *input;
         char *istatus;
         char *output;
-        char *ostatus;
 
         vloopbacks = fopen("/proc/video/vloopback/vloopbacks", "r");
 
@@ -76,11 +74,9 @@ static int v4l_open_vidpipe(void)
         while (fgets(buffer, sizeof(buffer), vloopbacks)) {
             if (strlen(buffer) > 1) {
                 buffer[strlen(buffer)-1] = 0;
-                loop = strtok(buffer, "\t");
                 input = strtok(NULL, "\t");
                 istatus = strtok(NULL, "\t");
                 output = strtok(NULL, "\t");
-                ostatus = strtok(NULL, "\t");
 
                 if (istatus[0] == '-') {
                     snprintf(pipepath, sizeof(pipepath), "/dev/%s", input);

